### PR TITLE
Add asserts which helps me debug issues with tests

### PR DESCRIPTION
**Pull request by kant2002**

### CODE REVIEW

**Comment by kant2002** [comment]
@ihhub can you take a look at this?

File: src/fheroes2/maps/maps.cpp [hunk 1]
```diff
@@ -210,6 +210,7 @@ bool Maps::isValidDirection( s32 from, int vector )
 
 Point Maps::GetPoint( s32 index )
 {
+    assert( world.w() != 0 );
```
Comment (ihhub) [inline src/fheroes2/maps/maps.cpp:213]: This method is called very often. We shouldn't do the check here.

Comment (kant2002) [reply src/fheroes2/maps/maps.cpp:213]: Isn't assert do nothing in Release builds?

Comment (ihhub) [reply src/fheroes2/maps/maps.cpp:213]: Right now we don't compile release version without assert as few users found some nasty bugs with assertion crashes.

Comment (ihhub) [reply src/fheroes2/maps/maps.cpp:213]: `NDEBUG` flag for Visual Studio is not set.

Comment (ihhub) [reply src/fheroes2/maps/maps.cpp:213]: @kant2002 , shall we revert this change for now?

**Comment by ihhub** [comment]
Well done, @kant2002 !

**Pull request merged**